buildkite/docker/ubuntu2204/Dockerfile

```diff
@@ -68,15 +68,6 @@ RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.c
     apt-get update -y && apt-get install google-cloud-sdk -y && \
     rm -rf /var/lib/apt/lists/*
 
-### Docker (for rules_docker)
```

Comment:
Do we need this for rules_oci?

Reply:
I suppose it would be nice to add various popular rulesets here, I think we should go through the list separately. For instance there's no testing of JS anymore

Reply:
Okay, sounds good.